modules/games.js

```diff
@@ -0,0 +1,318 @@
+      const embed = u.embed().setTitle(`Current Chess.com Games for ${name}`)
+        .setThumbnail("https://openclipart.org/image/800px/svg_to_png/275248/1_REY_Blanco_Pieza-Mural_.png");
+      let i = 0;
+      for (const game of games) {
+        embed.addField(`♙${getPlayer.exec(game.white)[1]} v ♟${getPlayer.exec(game.black)[1]}`, `Current Turn: ${(game.turn == "white" ? "♙" : "♟")}${getPlayer.exec(game[game.turn])[1]}\nMove By: ${moment(game.move_by).format("ddd h:mmA Z")}\n[[link]](${game.url})`, true);
+        if (++i == 25) break;
+      }
+      if (games.length == 0) {
+        result = await chessAPI.getPlayerStats(encodeURIComponent(name));
+        if (result) {
+          const daily = result.body["chess_daily"];
+          const tactics = result.body["tactics"];
+          const puzzle = result.body["puzzle_rush"]?.best;
+          const toTime = (time) => `<t:${time}:F>`;
+          const overall = daily?.record ? `Overall:\nWins: ${daily.record.win}\nLosses: ${daily.record.loss}\nDraws: ${daily.record.draw}\nTime Per Move: ${daily.record['time_per_move']}\n\n` : "";
+          const latest = daily?.last ? `Latest:\nRating: ${daily.last.rating}\nDate: ${toTime(daily.last.date)}\n\n` : "";
+          const best = daily?.best ? `Best:\nRating: ${daily.best.rating}\nDate: ${toTime(daily.best.date)}\n[Link](${daily.best.game})` : "";
+          embed.setTitle(`Chess.com Stats for ${name}`)
+            .addField("Chess Daily", (overall || latest || best) ? `${overall}${latest}${best}` : "No available stats")
+            .addField("Puzzle Rush", puzzle ? `Total Attempts: ${puzzle["total_attempts"]}\nHigh Score: ${puzzle.score}` : "No available stats")
+            .addField("Tactics", tactics ? `Highest Rating: ${tactics.highest.rating}\nLowest Rating: ${tactics.lowest.rating}` : "No available stats");
+          return inter.reply({ embeds: [embed] });
+        }
+      }
+      if (games.length == 0) embed.setDescription(`No active games found for ${name}`);
+      if (games.length > 25) embed.setDescription(`${name}'s first 25 active games:`);
+      else embed.setDescription(`${name}'s active games:`);
+      inter.reply({ embeds: [embed] });
+    } catch (error) {
+      if (error.message == "Not Found" && error.statusCode == 404) {
+        inter.reply({ content: `I couldn't find a profile for \`${name}\`.`, ephemeral: true });
+      } else { u.errorHandler(error, inter); }
+    }
+  } else {
+    inter.reply({ content: "I couldn't find a saved IGN for them.", ephemeral: true });
+  }
+}
+
+/** @param {discord.CommandInteraction} inter */
+async function elite(inter) {
+  const starSystem = inter.options.getString('system-name') ? await eliteAPI.getSystemInfo(inter.options.getString('system-name')) : null;
+  const embed = u.embed().setThumbnail("https://i.imgur.com/Ud8MOzY.png").setAuthor({ name: "EDSM", iconURL: "https://i.imgur.com/4NsBfKl.png" });
+  const getStatus = async () => {
```

Comment:
Let's break these out and define them outside of this function (named `eliteGetStatus` and similar) instead of making them as anonymous functions. The system can be passed in as a parameter for those that require it.